trajectory_sampler.py

```diff
@@ -67,6 +69,8 @@ def _is_sampling_possible(self, past_timedelta, future_timedelta, min_meters_per
         return True 
         
     def _get_actually_available_times(self, start_time, past_time, future_time):
+        # NOTE: this is a bit crufty, as everything about the times 
+        #   should be available in the index
```

Comment:
Not sure what to make of this comment.

Reply:
The logic of TrajectorySampler is explained in https://anitagraser.com/2018/12/15/movement-data-in-gis-18-creating-evaluation-data-for-trajectory-predictions/